autotest/utilities/test_gdalalg_vector_geom_set_type.py

```diff
@@ -320,3 +320,25 @@ def test_gdalalg_vector_geom_set_type_type_autocomplete():
         f"{gdal_path} completion gdal vector geom set-type --geometry-type GEOMETRYC"
```

Comment:
What does "GEOMETRYC" stand for? GeometryCollection shortened or what?

Reply:
yes that's for testing completion